gradle.properties

```diff
@@ -1,4 +1,8 @@
-org.gradle.jvmargs=-XX:+UseG1GC -Xmx512m -XX:MaxMetaspaceSize=512m
+org.gradle.jvmargs=-XX:+UseG1GC -Xmx512m -XX:MaxMetaspaceSize=512m --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
+  --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED \
```

Comment:
These shouldn't be required anymore with the latest google-java-format 1.15.0 release according to https://github.com/google/google-java-format/releases but for some reason those are still required on CI but not locally.